bootstrap/src/components/Page.module.scss

```diff
@@ -11,6 +11,7 @@
     flex: 1 0 auto;
     padding: $spacing * 2 + 56px $spacing $spacing * 2 $spacing;
     max-width: 1200px;
+    width: calc(100% - (40px));
```

Comment:
Why 40px? Is it 2 * $spacing? If so, please use the variable here

Reply:
Calc does not accept Sass variables as is. I just bumped into a post where they suggest to wrap it with #{}; therefore, it will look like this #{$spacing}.

Reply:
That is probably right :) Have you tried that?

Reply:
yep, it works well. Its already updated